blocks_vertical/data.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+goog.provide('Blockly.Blocks.data');
+
+goog.require('Blockly.Blocks');
+

```

Comment:
No empty between goog.require lines.  Keep the empty line between provide and require.